install.yaml

```diff
@@ -110,19 +123,28 @@ pre_install_actions:
       database:
         type: {{ regexReplaceAll "oracle-" (regexReplaceAll ":.*$" $dbtype "") "" }}
         version: {{ regexReplaceAll "^.*:" $dbtype "" }}
-
       docroot: {{ dig "web" "locations" "/" "root" "notfound" .platformapp }}
+      web_environment:
+      # TODO: Review which of these matters and which can be dummied up
+      - "PLATFORM_MOUNTS={{ range $key, $value := .platformapp.mounts }}{{ $key }} {{ end }}"
+      - "PLATFORM_APP_DIR=/var/www/html"
+      - PLATFORM_PROJECT_ENTROPY=$(echo $RANDOM | shasum -a 256 | awk '{print $1}')
+      # Consider commit hash for PLATFORM_TREE_ID
+      - "PLATFORM_TREE_ID=2dc356f2fea13ef683f9adc5fc5bd28e05ad992a"
```

Comment:
IMHO leaving this empty would be fine- I think it could be troublesome if the tree ID is set to something that doesn't match a remote environment. It can be approximated with a commit ID or even a git tree ID from the app's files, but an approximation may be more trouble than nothing.

Reply:
IIRC it can't be empty, which is why it's like this.

Reply:
It probably has to be set because of code like this (`$platformsh->treeId` would throw if it's not set):

https://github.com/platformsh/template-builder/blob/acb0d4b85a88b74a12bb903a48f102888a3d5110/templates/drupal9/files/web/sites/default/settings.platformsh.php#L114

but I believe an empty string could work too. If not, something like `placeholder` would work.

Reply:
What we have is essentially "placeholder" :)